Library/Formula/kibana.rb

```diff
@@ -0,0 +1,54 @@
+class Kibana < Formula
+  homepage "https://www.elastic.co/products/kibana"
+  url "https://github.com/elastic/kibana/archive/v4.0.2.tar.gz"
+  version "4.0.2"
+  sha256 "3792e423dd97a216e12b0bb1ee97cf75712b2f850422d3746ff076ce9e1f2e01"
+  head "https://github.com/elastic/kibana.git"
+
+  depends_on "node"
+
+  def install
+    prefix.install "src"
+
+    (etc/"kibana").mkpath
+    (var/"lib/kibana/plugins").mkpath
+
+    conf_path = etc/"kibana/kibana.yml"
+    cp "config/kibana.yml", conf_path unless conf_path.exist?
```

Comment:
`(etc/"kibana").install "config/kibana.yml" unless (etc/"kibana/kibana.yml").exist?` is possibly better.